firefox-ios/Client/ContentBlocker/ContentBlocker.swift

```diff
@@ -302,9 +304,10 @@ extension ContentBlocker {
         let defaults = UserDefaults.standard
         var hasChanged = false
 
+        let lists = RemoteDataType.contentBlockingLists
         for list in blocklists {
-            guard let path = Bundle.main.path(forResource: list, ofType: "json"),
-                  let newHash = calculateHash(forFileAtPath: path) else { continue }
+            guard let data = try? lists.loadLocalSettingsFileAsJSON(fileName: list) else { continue }
+            guard let newHash = calculateHash(for: data) else { continue }
```

Comment:
Not really something we can support right now, but once we figure out caching we can also get the file digest from RS for attachments.